### PR TITLE
[Opset] Opset16 initialization

### PR DESCRIPTION
**Pull request by PiotrKrzem**
### Details:
 - Initialization of OV Opset16

### Tickets:
 - 156876, Finalization 156877

### CODE REVIEW

**Review by mitruska — APPROVED** [review]

File: src/core/include/openvino/opsets/opset.hpp [hunk 1]
```diff
@@ -172,6 +172,11 @@ const OPENVINO_API OpSet& get_opset14();
  * @ingroup ov_opset_cpp_api
  */
 const OPENVINO_API OpSet& get_opset15();
+/**
+ * @brief Returns map of available opsets
```
Comment (t-jankowski) [inline src/core/include/openvino/opsets/opset.hpp:176]:
```suggestion
 * @brief Returns map of available operators
```

**Review by t-jankowski — CHANGES_REQUESTED** [review]

File: src/core/include/openvino/opsets/opset.hpp (hunk 1 above)
Comment (mitruska) [reply src/core/include/openvino/opsets/opset.hpp:176]: The requested change is not correct.
It should be:
```suggestion
 * @brief Returns opset16
```
But as it's a copy paste, of the previous `get_opset` description.
Let's align the comment for both, as a follow up PR.

Comment (t-jankowski) [reply src/core/include/openvino/opsets/opset.hpp:176]: Don't bother with separate PR for such a small change.

**Review by t-jankowski — APPROVED** [review]

File: src/core/include/openvino/opsets/opset.hpp (hunk 1 above)
Comment (PiotrKrzem) [reply src/core/include/openvino/opsets/opset.hpp:176]: Fixed in Identity Ref/Core

**Pull request merged**